src/librustc_data_structures/bitvec.rs

```diff
@@ -325,6 +333,14 @@ impl<R: Idx, C: Idx> SparseBitMatrix<R, C> {
         changed
     }
 
+    /// Merge a row, `from`, into the `into` row.
+    pub fn merge_into(&mut self, into: R, from: &SparseBitSet<C>) -> bool {
+        match self.vector.get_mut(into) {
```

Comment:
This should again use the `resize_to_elem` helper;

```rust
self.vector.resize_to_elem(into, || SparseBitSet::new());
self.vector[row].merge_into(from);
```

Reply:
Fixed this.